src/server/lib/templates/digest.tsx

```diff
@@ -21,6 +24,20 @@ interface NotificationContentProps {
   notification: FormattedNotification;
 }
 
+const styles = {
+  font: {
+    fontFamily: "Helvetica"
+  },
+  button: {
+    backgroundColor: assemblePalette.primary.navy,
+    padding: `12px 20px`,
+    border: "none",
+    color: "white",
+    borderRadius: 4,
+    marginBottom: 10
+  }
+};
```

Comment:
According to my view in mailtrap, the `makeStyles` hook didn't create static CSS that could be rendered via HTML. So I opted for a `styles` variable instead.

Reply:
I don't think `makeStyles` works using SSR. https://mui.com/material-ui/guides/server-rendering/